src/styles/_variables.scss

```diff
@@ -1,25 +1,51 @@
-// Box Design Language Theme Colors
+// Box Design Language Colors, confirm with design before extending
+// @see https://gallery.io/projects/MCHbtQVoQ2HCZTdZ0F79-fQJ/files/MCEJu8Y2hyDScSTYfw7hxWcA-L66cSQvSj4
+
+// Box/Primary Blues
 $bdl-box-blue: #0061d5 !default;
-$bdl-secondary-blue: #003c84 !default;
+$bdl-box-blue-50: #619de5 !default;
+$bdl-box-blue-05: #f2f7fd !default;
+
+// Box/Primary Grays
+$bdl-gray: #222 !default;
+$bdl-gray-62: #767676 !default;
```

Comment:
It looks like in some places we sort ascending and others descending. Maybe standardize on the former?

Reply:
Could we update the colors.md to show the new palette / categories documented here?